src/locales/id.json

```diff
@@ -0,0 +1,95 @@
+    "bringToFront": "Bawa ke Depan",
+    "sendBackward": "Kirim Mundur",
+    "delete": "Hapus",
+    "copyStyles": "Salin Style",
+    "pasteStyles": "Tempel Style",
+    "stroke": "Stroke",
+    "background": "Latar",
+    "fill": "Fill",
+    "strokeWidth": "Lebar Stroke",
+    "sloppiness": "Sloppiness",
+    "opacity": "Opacity",
+    "fontSize": "Ukuran Font",
+    "fontFamily": "Font Family",
+    "onlySelected": "Hanya yang Dipilih",
+    "withBackground": "Dengan Latar",
+    "handDrawn": "Hand-Drawn",
+    "normal": "Normal",
+    "code": "Kode",
+    "small": "Kecil",
+    "medium": "Medium",
+    "large": "Besar",
+    "veryLarge": "Sangat Besar",
+    "solid": "Solid",
+    "hachure": "Hachure",
+    "crossHatch": "Cross-Hatch",
+    "thin": "Lembut",
+    "bold": "Tebal",
+    "extraBold": "Sangat Tebal",
+    "architect": "Arsitek",
+    "artist": "Artis",
+    "cartoonist": "Cartoonist",
+    "fileTitle": "Judul Berkas",
+    "colorPicker": "Pilihan Warna",
+    "canvasBackground": "Latar Kanvas",
+    "drawingCanvas": "Drawing Canvas",
+    "layers": "Lapisan",
+    "language": "Bahasa"
+  },
+  "buttons": {
+    "clearReset": "Setel Ulang Kanvas",
+    "export": "Ekspor",
+    "exportToPng": "Expor ke PNG",
+    "exportToSvg": "Expor ke SVG",
+    "copyToClipboard": "Salin ke Papan Klip",
+    "save": "Simpan",
+    "load": "Muat",
+    "getShareableLink": "Buat Tautan yang Bisa Dibagian",
+    "close": "Tutup",
+    "selectLanguage": "Pilih Bahasa",
+    "scrollBackToContent": "Scroll back to content",
+    "zoomIn": "Besarkan",
+    "zoomOut": "Kecilkan",
+    "resetZoom": "Reset Pembesaran",
+    "menu": "Menu",
+    "done": "Selesai",
+    "edit": "Edit",
+    "undo": "Undo",
+    "redo": "Redo"
+  },
+  "alerts": {
+    "clearReset": "Ini akan menghapus semua yang ada dikanvas. Apakah kamu yakin ?",
+    "couldNotCreateShareableLink": "Tidak bisa membuat tautan yang bisa dibagikan",
+    "importBackendFailed": "Gagal mengimpor dari backend",
+    "cannotExportEmptyCanvas": "Tidak bisa mengekspor kanvas kosong",
+    "couldNotCopyToClipboard": "Tidak bisa menyalin ke papan klip. Coba gunakan Browser Chrome",
+    "uploadedSecurly": "Pengunggahan ini telah diamankan menggunakan enkripsi end-to-end, artinya server Excalidraw dan pihak ketiga tidak data membaca nya"
+  },
+  "toolBar": {
+    "selection": "Pilihan",
+    "rectangle": "Persegi",
+    "diamond": "Berlian",
+    "ellipse": "Elips",
+    "arrow": "Panah",
+    "line": "Garis",
+    "text": "Teks",
+    "lock": "Biarkan alat yang dipilih aktif setelah menggambar"
+  },
+  "headings": {
+    "canvasActions": "Aksi Kanvas",
+    "selectedShapeActions": "Selected shape actions | Aksi bentuk yang dipilih",
+    "shapes": "Bentuk"
+  },
+  "hints": {
+    "linearElement": "Klik untuk memulai banyak poin, seret untuk satu baris",
+    "linearElementMulti": "Press Escape or Enter to finish | Tekan Escape atau Enter untuk menyelesaikan",
```

Comment:
The “Press Escape or enter to finish” part can be removed, correct?

Reply:
Oh i see, that is i used for placeholder but i forgot to remove that